src/bin/__test__/cli.test.ts

```diff
@@ -0,0 +1,37 @@
+import run from '../cli'
+
+afterEach(() => {
+  jest.resetAllMocks()
+})
+
+test('cli prints help', () => {
+  const logSpy = jest.spyOn(global.console, 'log').mockImplementation(jest.fn())
```

Comment:
`.mockImplementation(() => {})` might be clearer if you're just wanting to prevent calling through to the original implmentation.